### PR TITLE
Update ais-vd_pi to 1.8.1 for O 5.8

### PR DESCRIPTION
**Pull request by Hakansv**

### CODE REVIEW

**Comment by Hakansv** [comment]
All Url validation passed
Any hint what I have to with this?
xmllint error(s)
Error: Process completed with exit code 1.

**Comment by bdbcat** [comment]
Problem with ais-vd-1.8-darwin-wx315-10.13.6.xml.

From the log:
Processing file: metadata/ais-vd-1.8-darwin-wx315-10.13.6.xml
metadata/ais-vd-1.8-darwin-wx315-10.13.6.xml:22: element target: Schemas validity error : Element 'target': [facet 'enumeration'] The value 'darwin-wx315' is not an element of the set {'all', 'android-arm64', 'android-arm64-v8a', 'android-armeabi-v7a', 'android-armhf', 'darwin', 'darwin-arm64', 'darwin-wx32', 'debian-armhf', 'debian-wx32-armhf', 'debian-gtk3-armhf', 'debian-x86_64', 'debian-wx32-x86_64', 'debian-arm64', 'debian-wx32-arm64', 'flatpak-aarch64', 'flatpak-x86_64', 'mingw', 'mingw-x86_64', 'msvc', 'msvc-64', 'msvc-wx32', 'raspbian-armhf', 'ubuntu-armhf', 'ubuntu-gtk3-armhf', 'ubuntu-gtk3-x86_64', 'ubuntu-x86_64'}.
metadata/ais-vd-1.8-darwin-wx315-10.13.6.xml fails to validate

**Comment by rgleason** [comment]
I see ais-vd 1.0.8 in the prod repos. so it isn't that.
https://cloudsmith.io/~opencpn/repos/ais-vd-prod/packages/

**Comment by Hakansv** [comment]
No the new ais-vd 1.8.1 is not in the production repo so something failed.
The log says validation failed due to ais-vd-1.8-darwin-wx315-10 was present but not in the list so there seems to have been one extra file included?
It's somewhat hard to follow since the ais-vd version is set and tagged to 1.8.1 but all files are called 1.8 only? Long time since i did this process last time so I may have forgot something?
I may redo everything and update to version 1.9 since only two levels seems to be used?

**Comment by rgleason** [comment]
I am sketchy about the SD template too, but did your template update run and complete properly? You need to run that twice, first to update the template and then again to update the files. I think incrementing to 1.9 is a very good idea as it is intended for Opencpn 5.7.1 and wxWidgets 3.2.1 in most cases.

**Comment by Hakansv** [comment]
Hi Rick. 
My intention in update from 1.8.0 to 1.8.1 was exactly that.  Still 1.8 since no change to plugin function. 1.8.1 to indicate version for wx321.
The "Shipdriver" update to sd3.2.0 on the plugin itself worked fine
And checking [CS alpha ](https://cloudsmith.io/~opencpn/repos/ais-vd-alpha/packages/) all files are there although called 1.8 but the description says 1.8.1.
It was the git/plugin update and push to OpenCPN/plugins that failed. I'm trying to again learn and follow the advises but probably lost somewhere.
Just now I don't remember how the repo /plugins know which plugin (ais-vd) I'm about to push and are there files I've to edit for a nice plugin version designation, e.g. "ais-vd 1.8.1"  and nothing more.

**Comment by rgleason** [comment]
Hakan, did you delete your plugins "auto" branch? and setup your plugins correctly before pushing to git?
I have had troubles with this a number of times and resort to just doing it manually with jon's nice script in a windows bash prompt  using  download_xml_bash.sh ais-vd 1.8.1 opencp alpha(?)  after making sure your beta or master branch is current with upstream.

Maybe Mike will notice and be able to help.
